### PR TITLE
Enhancement / fake ancient

### LINKED ISSUE #11: Add Fake Ancient
Add [ancient](https://github.com/faker-ruby/faker/blob/master/doc/default/ancient.md), which include: `God`, `Primordial`, `Titan`, and `Hero`.

### PR DESCRIPTION
**Pull request by Caps-Looking**
Closes #11 

## Description
Creation of a wrapper of fake ancients.

## Technical Details
- Changed `FakeContext` to add `fakeAncient`.
- Added `FakeAncient` class to provide the `fakeAncient`.
- Changed `FakerTypeAliases` to add `FakerAncient` alias.
- Changed `FakerContextTest` to add `fakeAncient` test.
- Added `FakeAncientTest` to test `FakeAncient`.

## Code Samples
```kotlin
fun main() {
    val fakeAncient: FakeAncient(god="God", primordial="Primordial", titan="Titan", hero="Hero")
    fakek { println("Ancient: God - ${fakeAncient.god}, Primordial - ${fakeAncient.primordial}, Titan - ${fakeAncient.titan}, Hero - ${fakeAncient.hero}") }
}
```

Reviewers: @CodyEngel

### CODE REVIEW

File: src/main/kotlin/dev/fakek/fakes/FakeAncient.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package dev.fakek.fakes
+
+/**
+ * FakeName provides a fake ancient which includes a god, primordial, titan, and hero.
```
Comment (CodyEngel) [inline src/main/kotlin/dev/fakek/fakes/FakeAncient.kt:4]: Should this be `FakeAncient`?

File: src/main/kotlin/dev/fakek/FakeContext.kt [hunk 2]
```diff
@@ -38,6 +39,11 @@ class FakeContext(private val faker: Faker = Faker.instance()) {
      */
     val fakeAddress by lazy { FakeAddress(fakerAddress) }
 
+    /**
+     * Provides a [FakeAncient].
+     */
+    val fakeAncient by lazy { FakeAncient(fakerAncient) }
```
Comment (CodyEngel) [inline src/main/kotlin/dev/fakek/FakeContext.kt:45]: Thanks for keeping this in alphabetical order 🙌

**Review by CodyEngel — CHANGES_REQUESTED** [review]
Documentation appears to have a copy/paste error but otherwise changes look good, thank @Caps-Looking.

File: src/main/kotlin/dev/fakek/fakes/FakeAncient.kt (hunk 1 above)
Comment (Caps-Looking) [reply src/main/kotlin/dev/fakek/fakes/FakeAncient.kt]: Oops, sorry 😅

**Review by CodyEngel — APPROVED** [review]
LGTM 👍

**Pull request merged**